eng/pipelines/templates/jobs/ci.yml

```diff
@@ -332,9 +331,14 @@ jobs:
           ServiceDirectory: "template"
           TestPipeline: ${{ parameters.TestPipeline }}
 
-      - template: /eng/common/pipelines/templates/steps/verify-readme.yml
-        parameters:
-          ScanPath: $(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}
+      - ${{ if and(eq(variables['Build.Reason'], 'PullRequest'), eq(parameters.ServiceDirectory, 'auto')) }}:
```

Comment:
Lets try to switch to using verify-readmes.yml for all cases.

Reply:
@weshaggard I'm not changing these as there is no one size fits all right now. Even if I wanted to delay the Analyze job until after the Build job, which I don't right now, that's not going to remove all of the `if PullRequest` checks. I can create an issue to look at this later.